R/utilities-pk-analysis.R

```diff
@@ -95,5 +95,16 @@ pkAnalysesToDataFrame <- function(pkAnalyses) {
     }
   )
 
-  return(dplyr::as_tibble(dataFrame))
+  # consistently return a (classical) data frame
+  return(df)
```

Comment:
`df` is not present, here it is called `dataFrame`

Reply:
Strange that tests didn't fail.

Renamed `dataFrame` to `df` to be consistent across functions.